app/models/leader.rb

```diff
@@ -0,0 +1,5 @@
+class Leader < ApplicationRecord
+	validates :email, :uniqueness => { :case_sensitive => false }
```

Comment:
Hm interesting. Usually I put all of the validations for a single column in one line, e.g.

```ruby
validates(:email, {
  :presence => true,
  :uniqueness => { :case_sensitive => false },
})
```

But I kind of like having each one on its own line, maybe that will be easier for students to understand?